.github/workflows/main.yml

```diff
@@ -78,15 +78,11 @@ jobs:
           node-version: '16'
 
       - name: Install app dependencies
-        run: npm install
+        run: npm install --force
```

Comment:
Perhaps using `npm install --legacy-peer-deps` would be better for future upgrades.